aws/defs.bzl

```diff
@@ -1 +1,54 @@
 "Public API re-exports"
+
+load("@rules_oci//oci:defs.bzl", "oci_image")
+load("//aws/private:py_lambda.bzl", "py_lambda_tars")
+load("@rules_python//python:defs.bzl", "py_binary")
+
+def aws_py_lambda(name, entry_point = "lambda_function.py", deps = [], base = "@aws_lambda_python"):
```

Comment:
Since Lambdas can be OCI or not, should we specify this is for OCI only?

Reply:
They can be OCI images or Zip files. The latter is already a TODO below.
(Docker isn't relevant here because we killed rules_docker, and `oci_push` works with AWS ECR)